### PR TITLE
Adding automation new issues in project board

### PR DESCRIPTION
**Pull request by mijovic**

### CODE REVIEW

File: .github/workflows/assigned_issues.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Move new issues into "New issues" in "Solidity" project board
```
Comment (axic) [inline .github/workflows/assigned_issues.yml]: This seems to be for assignment?

Comment (mijovic) [reply .github/workflows/assigned_issues.yml]: Copy paste error... Fixing now

File: .github/workflows/stale_issues.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Issue has been inactive for too long. Consider closing or moving to icebox. This message is automated.'
+        stale-issue-label: 'no-issue-activity'
```
Comment (mijovic) [inline .github/workflows/stale_issues.yml]: Needs to exclude issues that are already in Icebox

File: .github/workflows/assigned_issues.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  automate-project-columns:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: alex-page/github-project-automation-plus@v0.2.3
+        with:
+          project: Solidity
+          column: In progress
+          repo-token: ${{ secrets.ASSIGNED_ISSUES_TOKEN }}
```
Comment (mijovic) [inline .github/workflows/assigned_issues.yml]: We can use `GITHUB_TOKEN` as well

**Comment by mijovic** [comment]
It is updated to work with new issues only

**Comment by chriseth** [comment]
Will this "just work" once it is merged?

**Comment by mijovic** [comment]
> Will this "just work" once it is merged?

It should "just work". But let me check once more on my repo so I can be 100% sure.

**Comment by mijovic** [comment]
Changed to different github action, as the one I used for first needed more permissions than `GITHUB_TOKEN` is giving.
You can find here more about `GITHUB_TOKEN` https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

Updated PR and it will sort issues to `Solidity` project automatically once merged (and default column is `New issues`).

**Comment by chriseth** [comment]
Looking at skeet70/default-project-board-action I find it weird that it has to perform additional requests to retrieve the issue and project id. I'm wondering if we were better of just writing those scripts ourselves...

**Comment by mijovic** [comment]
> Looking at skeet70/default-project-board-action I find it weird that it has to perform additional requests to retrieve the issue and project id. I'm wondering if we were better of just writing those scripts ourselves...

I think yes, as it is really simple thing. That will allow us to do more custom things later.
But than, where it should be done. I think solidity repo is not the place where we want this code :)

**Comment by chriseth** [comment]
I'm mainly worried that we are exposing the github token to all the npm dependencies this pulls in.

**Comment by leonardoalt** [comment]
Was there a decision whether to use this or write our own?

**Comment by chriseth** [comment]
I would feel more comfortable with writing our own - I don't think it is more complicated than the scripts we already have, and it can be tweaked more easily.

**Comment by mijovic** [comment]
> I would feel more comfortable with writing our own - I don't think it is more complicated than the scripts we already have, and it can be tweaked more easily.

Let's close this one than and start writing our own.

**Pull request closed without merging**